### PR TITLE
Remove markets live

### PR DESCRIPTION
**Pull request by ker-an**
This PR removes any references to the Markets Live API as `alphaville-blogs` no longer asks `alphaville-es-interface-service` for the Markets Live content. See https://github.com/Financial-Times/alphaville-blogs/pull/59. 

This work is related to the ongoing decommissioning of Markets Live, see https://github.com/Financial-Times/next/issues/400.

Oh and I also tacked on the `comment-creation-service` removal because it was being used in a Markets Live file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,6 @@
   "license": "ISC",
   "dependencies": {
     "@financial-times/n-es-client": "^1.0.0-beta.6",
-    "alphaville-marketslive-wordpress-api": "github:financial-times/alphaville-marketslive-wordpress-api#0.4.0",
```
Comment (sjparkinson) [inline package.json:18]: 🙌

**Comment by sjparkinson** [comment]
One suggestion I'd have is to use `heroku logs --app av2-es-interface-service` and `grep` to validate that we are not making any requests to the routes being deleted.

I'm 100% sure they are not, but checking in production is a good way to verify.

**Review by sjparkinson — APPROVED** [review]

**Comment by ker-an** [comment]
@sjparkinson yes, great idea. I will double check that we're not making any requests to those routes.

**Comment by ker-an** [comment]
@sjparkinson I checked the Heroku logs and couldn't find any requests to the `/marketslive` route. They all seem to be hitting `/gtg`, `/article` and `/topic` routes.

**Pull request merged**